src/main/java/v/hryhoryk/onlinebookstore/controller/BookController.java

```diff
@@ -44,6 +45,7 @@ public class BookController {
                     description = "Not found - no books are available")
     })
     @ResponseStatus(HttpStatus.OK)
+    @PreAuthorize("hasAnyRole('ADMIN', 'USER')")
```

Comment:
shouldn't admin user also have a role `USER`?

or you design your app in a way that admin does not have same priviliges as user has?